wolox-books/src/app/screens/list-books/list-books.component.ts

```diff
@@ -1,12 +1,29 @@
 import { Component, OnInit } from '@angular/core';
 
+import { ListBooksService } from 'src/app/services/list-books.service';
+
 @Component({
   selector: 'app-list-books',
   templateUrl: './list-books.component.html',
   styleUrls: ['./list-books.component.scss']
 })
-export class ListBooksComponent {
+export class ListBooksComponent implements OnInit {
+
+  books: any;
+
+  constructor(private listBooksService: ListBooksService) { }
+
+  ngOnInit() {
+    this.getLocalStorage();
+  }
+
+  getLocalStorage() {
+    const valueLocalStorage = localStorage.getItem('token').replace(/['"]+/g, '');
+    this.listBooks(valueLocalStorage)
+  }
 
-  constructor() { }
+  listBooks(valueLocalStorage) {
```

Comment:
chande the variable name, what is valueLocalStorage, if it's a token, use a token, if it's userId use userId